frontend/src/assets/css/auth.scss

```diff
@@ -1,38 +1,19 @@
-p,
```

Comment:
auth.scss should be a module.scss file next to a component which he style.

Reply:
ok. auth.scss was from previous task not by me, but I'll do it